packages/svelte-ux/src/lib/utils/number.ts

```diff
@@ -1,81 +1,93 @@
-import { format as d3Format } from 'd3-format';
+import { getFormatNumberOptions } from '$lib/components/setup';
+
+export type FormatNumberStyle =
+  | 'decimal' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'currency' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'percent' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'unit' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'none'
+  | 'integer'
+  | 'percentRound'
+  | 'metric';
+
+export type FormatNumberOptions = Intl.NumberFormatOptions & {
+  style?: FormatNumberStyle;
+  locales?: string | undefined;
```

Comment:
I was thinking this should be called `locale` instead of `locales`, but the argument is called that.  TIL 😁

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#using_locales

Reply:
hahaha, I did 20 go & back on this one, and then saw why it could be an array & fallback & co.
Smarter ppl did this, let's follow 😉